### DISCUSSION TITLE
Delete record with Related records

### DISCUSSION THREAD

**Discussion by bodawui** [burkybang/SuiteAdvanced-Feedback#114] [Ideas]
Hello,

I'm very pissed of when I need to delete record and I'm not able to do so due to related records and then duing masive analysis of those.

mostly it is related to ARM module, when you have all of those Revenue records.

I do understand that it is dangerous to use that, but for development of batch processing (like deleting 20 records) it is pain in the ass.

would be very and very usefull to have such "delete with related records" button

Thanks a lot for considering this idea

**Comment by burkybang** [comment]
Yeah.... as you said, I think it would be too dangerous to have such a feature. I don't want to be liable for helping to ruin someone's real data.
